sdk/python/feast/infra/online_stores/redis.py

```diff
@@ -68,6 +68,9 @@ class RedisOnlineStoreConfig(FeastConfigBaseModel):
     """Connection string containing the host, port, and configuration parameters for Redis
      format: host:port,parameter1,parameter2 eg. redis:6379,db=0 """
 
+    key_ttl_seconds: int = None
```

Comment:
nit: mark the int as optional too then? i.e. Optional[int]

Reply:
ah yeah good point!